templates/repo/migrate/migrating.tmpl

```diff
@@ -7,11 +7,16 @@
 				{{template "base/alert" .}}
 				<div class="home">
 					<div class="ui stackable middle very relaxed page grid">
-						<div id="repo_migrating" class="sixteen wide center aligned centered column" repo="{{.Repo.Repository.FullName}}">
+						<div id="repo_migrating" class="sixteen wide center aligned centered column" task="{{.MigrateTask.ID}}">
 							<div>
 								<img src="{{StaticUrlPrefix}}/img/loading.png"/>
 							</div>
 						</div>
+						<div id="repo_migrating_failed_image" class="sixteen wide center aligned centered column">
+							<div>
+								<img src="{{StaticUrlPrefix}}/img/failed.png"/>
```

Comment:
Is this a new image?

Reply:
yes. It's simply the gitea cup upside down